.github/workflows/tests.yml

```diff
@@ -83,7 +83,8 @@ jobs:
             - name: Check commit message
               if: github.event_name == 'pull_request'
               run: |
-                  git-tools/commit-msg-hook-range.sh $(git rev-list --merges -n 1 HEAD^1)..$GITHUB_SHA
+                  git fetch origin $GITHUB_BASE_REF &> /dev/null
+                  git-tools/commit-msg-hook-range.sh $(git rev-parse origin/$GITHUB_BASE_REF)..$GITHUB_SHA
```

Comment:
Does this work with rebases? If I'm not mistaken base ref is not guaranteed to be refreshed, so we might still need the `merges` in there. Maybe.

Reply:
`$GITHUB_BASE_REF` resolves to the branch name and hence should be the branch against which the pull request was created. For `3.3.x` this will run `git fetch origin 3.3.x` and then get the head commit hash with `git rev-parse origin/$GITHUB_BASE_REF`